src/actions/redux-action-creator/redux-action-creator.js

```diff
@@ -0,0 +1,2 @@
+export const reduxActionCreator = actionType => data => ({ type: actionType, data });
+export const reduxActionCreatorNoData = actionType => () => ({ type: actionType });
```

Comment:
1. You don't need `reduxActionCreatorNoData` because you can keep data undefined. There shouldn't be a problem with that.
2. Can we name `reduxActionCreator` something like `createReduxAction`. That reads a little better as a function.

Reply:
Finished